config/database.yml

```diff
@@ -4,22 +4,36 @@
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
 #
+# MySQL version 2.x
+#   gem install mysql2
+#
+#   Ensure the MySQL 2 gem is defined in your Gemfile
+#   gem 'mysql2'
+
 default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
+  adapter: mysql2
+  encoding: utf8
+  database: predeploychecker
+  username: predeploychecker
+  password: <%= ENV['PDC_DATABASE_PASSWORD'] %>
+  port: 3306
 
 development:
   <<: *default
-  database: data/db/development.sqlite3
+  host: staging-predeploychecker.czrldetpmste.us-east-1.rds.amazonaws.com
```

Comment:
This can probably stay as sqlite for the `development` Rails environment.  The Rails development environment is run locally on a laptop, so it won't have access to anything inside an AWS VPC.  I suggest leaving the `default` section as it was and instead make the RDS changes in the `production` section below.

@jebentier agree?  Or should we move the dev environment to a local MySQL instance to avoid any possible database implementation inconsistencies?

Reply:
It would actually be best to run this entirely as MySQL since we’re changing production. Migration and implementation bugs can come from using split environments like this and it’s better to test and develop in a production like environment.

Reply:
As Christian mentioned, this host is actually the one we want to use in the staging cluster, but that container runs the production rails environment. Some notes on this setup as a whole:

1. All the adapters should be converted to use `mysql2`
2. The username/password should be set in the default configuration using ENV variables with sane defaults that is expected to be used for local development (i think for us that's usually username `rr-dev` and password `dev`)
3. `database` should be set to `predeploychecker_dev` and `predeploychecker_test` for the development and test environments respectively 

@christian it's more common now-a-days to use `url: ENV['DATABASE_URL']` for setting up the production environment's database adapter. How to you feel about doing this for little applications like this one?  The ENV variable would be populated from a k8s secret.  In this case it would end up being `mysql2://<username>:<password>@staging-predeploychecker.czrldetpmste.us-east-1.rds.amazonaws.com:3306/predeploychecker`

Reply:
> How to you feel about doing this for little applications like this one?
@jebentier Sounds great to me.  I'm all for following conventions!